geometry/CMakeLists.txt

```diff
@@ -8,18 +8,25 @@ set(
   angles.hpp
   any_rect2d.hpp
   avg_vector.hpp
+  bbox.cpp
```

Comment:
These names don't seem descriptive enough. While "bbox" may pass as a common term, "cbox" and "dbox" are surely too narrow. Please, either rename the files or create a subdirectory for them and an explanatory comment.

Reply:
Please, suggest a name for cbox. I'm going to rename bbox to a bounding box and dbox to a diamond box.